### DISCUSSION TITLE
Send file size limit ?

### DISCUSSION THREAD

**Discussion by centropaghe** [dani-garcia/vaultwarden#4285] [Q&A]
Hi, I try to upload a file to a SEND. The file is 5 or 1 mb...The form says that the limit is 500mb
This is the error

<html> <head><title>413 Request Entity Too Large</title></head> <body> <center><h1>413 Request Entity Too Large</h1></center> <hr><center>nginx/1.19.3</center> </body> </html>

**Comment by stefan0xC** [comment]
I think you forgot to configure nginx to allow bigger uploads (it defaults to 1MB if I recall correctly):

```nginx
client_max_body_size 525M;
```

 Cf. https://github.com/dani-garcia/vaultwarden/wiki/Proxy-examples and https://nginx.org/en/docs/http/ngx_http_core_module.html#client_max_body_size
